### ISSUE TITLE
It is not possible to run this development environment on Apple Silicon

### ISSUE THREAD

**Issue by slifty** [PermanentOrg/devenv#37]
Apple has discontinued use of intel chips and has replaced their hardware to use an [Arm chipset](https://en.wikipedia.org/wiki/Apple_silicon).

VirtualBox is a "x86 and AMD64/Intel64 virtualization product" -- which unfortunately means it cannot run on new macs.

[This locked and stickied post](https://forums.virtualbox.org/viewtopic.php?t=98742) indicates there will *NEVER* be VirtualBox support for Arm.

**Comment by dyerjos** [comment]
Have you considered switching over to Docker? My company switched from Vagrant to Docker about 3 years ago and it was a very smooth transition. Just one command to the build a container on a Mac, Windows, or Linux machine and it's good to go. Not to mention that for an open-source project it would be great to lower the barrier to new people setting up their local dev environments.

**Comment by slifty** [comment]
@dyerjos you're absolutely right -- there has been some discussion about whether the right next step would be Docker or whether trying to remove the need for any form of container would be the best approach (e.g. transitioning away from a PHP backend to a typescript backend that can be run directly on a dev machine.

I definitely agree there's merit to Docker (though Docker has its own set of awkwardness when it comes to OSX, due to the closed source nature of Docker Desktop).

For someone like yourself, though the biggest blocker is the fact that the backend software isn't yet published in the first place!  I'm optimistic the project can get there.

**Comment by dyerjos** [comment]
Nice to meet you too! I interviewed with Cecilia last week Friday and thought I'd peruse the open source repo's over the weekend.

Docker does have some hiccups every now and then on a Mac. I've had many docker errors resolve with `docker system prune` and ideally I should never have to delete a build cache like that. Compared to Vagrant though I love Docker. VirtualBox just makes my mac sound like a jet engine but docker runs very smooth. 

Another reason I mention docker is it's one step closer to using Kubernetes too where you could even utilize kubernetes secrets for stuff like env variables (api tokens, etc) and you could scale things up easier.

**Comment by kfogel** [comment]
Hey, @dyerjos, I can never figure out how to PM someone on different platforms, but if I wanted to get in contact with you re contract work, how would I do that?  (I'm `kfogel {_AT_} opentechstrategies.com` if old-fashioned email works.)

**Comment by slifty** [comment]
We now have a docker-based development environment option so problem solved 🎉
